### PR TITLE
Set-DbaAgentOperator : Enhance Failsafe Notification Method Handling

### PR DESCRIPTION
**Pull request by howarthcd**
## Type of Change

 - [] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Purpose

Enhances Set-DbaAgentOperator to enable multiple failsafe operator notification methods to be specified.
### Approach

- Modifies the FailsafeNotificationMethod input parameter to be a string array.
- Adds validation for the input value(s).
- Enumerates the input value(s) and sets the notification method accordingly.

### Commands to test

#Valid - Default is 'NotifyEmail'
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator

#Valid
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod None
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod Pager
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod Pager,NotifyEmail
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod NotifyAll

#Invalid
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod NotifyAll,None
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod NotifyEmail,None
Set-DbaAgentOperator -SqlInstance localhost -Operator DBA -EmailAddress test@mydomain.com -IsFailsafeOperator -FailsafeNotificationMethod Pager,NotifyAll

### CODE REVIEW

**Comment by potatoqualitee** [comment]
so fast! great job 🥳  I'll merge after @andreasjordan approves.

**Review by andreasjordan — APPROVED** [review]
Looks good!

**Comment by potatoqualitee** [comment]
great! I'll fix the formatting then merge 🥳

**Pull request merged**
